jtabba/CDW-1
# Http requests

## Type of change

[x] New feature
<br />
[] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure


## Description
Setup HTTP endpoints with Axios Client, There is an issue with the UUID implementation in the userService module which i believe is not continuing on to production.


## Ticket link
https://github.com/jtabba/CDW-1/pull/31


## Part of a back end implementation?
[] No
<br />
[x] [Yes]()  


## How to test
<!-- If your ticket is testable outline the test steps
    Example:
        1) Checkout to this branch and run the app
        2) npm run start on front and back ends. 
        3) refresh page see data returned from Database in the console. 
        4) you can add a new user object in the post request
